build/steps/build-windows.yml

```diff
@@ -70,23 +65,9 @@ jobs:
       inputs:
         restoreSolution: ${{ parameters.slnPath }}
 
-    - task: MSBuild@1
```

Comment:
@rmarinho So I moved all these builds into cake, because of that whole issue where branches get merged into the head branch which causes the yaml to be inconsistent with the build.

I'd like to just start moving whatever we can on the YML into cake so that the build can just more easily travel with the code. 

Plus this let's me run the same build tasks for generating the nugets locally as we will run on the servers.